From: Marcin Owsiany <marcin@owsiany.pl>
Subject: [PATCH] debian/unqualify-xmotd-in-manpage

Remove /usr/local/bin/ from the manpage content.
The path is incorrect and useless for the Debian installation.

Signed-off-by: Marcin Owsiany <marcin@owsiany.pl>

---
 xmotd.8 | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/xmotd.8 b/xmotd.8
index 6a54563..d5adda9 100644
--- a/xmotd.8
+++ b/xmotd.8
@@ -5,17 +5,17 @@ xmotd \- message-of-the-day browser for X (and dumb terminals,
 VT100, etc.)
 
 .SH SYNOPSIS
-.B /usr/local/bin/xmotd 
+.B xmotd 
 [\fBX-toolkit options\fP] [\fBxmotd-options\fP] \fIfile\fP
 [\fIfile2\fP ... ]
 
-.B /usr/local/bin/xmotd 
+.B xmotd 
 [\fBX-toolkit options\fP] [\fBxmotd-options\fP] \fIdirectory\fP
 
 .PP
 or (in text-mode)
 
-.B /usr/local/bin/xmotd
+.B xmotd
 [\fB-stampfile\fP \fIstamp-file\fP] [\fB-wakeup\fP \fIsleep-period\fP]
 \fIfile\fP [\fIfile2\fP ...]
 
@@ -297,7 +297,7 @@ option. This example (typed as a single line) illustrates how
 
 .nf
 
-    /usr/local/bin/xmotd -always \\
+    xmotd -always \\
         -xrm "*title.label: Top 10 Disk Hogs\\n As of midnight\\n " \\
         -xrm "*title.foreground: yellow" \\
         -xrm "*form.background: red" \\
-- 
tg: (d3dafc5..) debian/unqualify-xmotd-in-manpage (depends on: upstream)
